TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -51,13 +63,34 @@ class CustomNavigationController: UINavigationController, UINavigationController
             Analytics.shared.log(payload)
         }
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // Add Notification Observers
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(reachabilityChanged(notif:)),
+                                               name: .reachabilityChanged,
+                                               object: reachability)
+        guard let reach = reachability else {return}
```

Comment:
can we do `guard let reachability = reachability` just so theres no confusion as to what `reach` is